tests/od/main.c

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Tests od module.
+ *
+ * @author      Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "od.h"
+
+#define CALL(fn)    puts(#fn); fn
+
+static const char short_str[] = "AB";
+static const char long_str[] = "\xff,a\xff.bcdefghijklmnop";
+
+int main(void)
+{
+    CALL(od(short_str, sizeof(short_str), OD_WIDTH_DEFAULT, 0));
+
+    CALL(od(long_str, sizeof(long_str), OD_WIDTH_DEFAULT, 0));
+    CALL(od(long_str, sizeof(long_str), 4, 0));
+    CALL(od(long_str, sizeof(long_str), 3, 0));
+    CALL(od(long_str, sizeof(long_str), 8, 0));
+
+
```

Comment:
I'm trying to imagine a pattern for the use of one or two empty lines between the code blocks.
Could this be some reserved space for the comments explaining what is tested by the different "cases"? ;-)